iphone/Maps/Classes/CustomViews/Login/MWMAuthorizationOSMLoginViewController.mm

```diff
@@ -138,6 +139,9 @@ - (IBAction)login
       catch (exception const & ex)
       {
         LOG(LWARNING, ("Error login", ex.what()));
+        [Statistics logEvent:@"Editor_Reg_request_result" withParameters:@{kStatIsSuccess : kStatNo,
```

Comment:
Ключи должны быть одинаковые, у них разные значения - осм, фб и гугл.

> On 25.3.2016, at 21.33, Dmitry Yunitsky <notifications@github.com> wrote:
> 
> In iphone/Maps/Classes/CustomViews/Login/MWMAuthorizationOSMLoginViewController.mm:
> 
> > @@ -138,6 +139,9 @@ - (IBAction)login
> >        catch (exception const & ex)
> >        {
> >          LOG(LWARNING, ("Error login", ex.what()));
> > +        [Statistics logEvent:@"Editor_Reg_request_result" withParameters:@{kStatIsSuccess : kStatNo,
> ?? Я о том, что в айос для авторизации и регистрации в этом реквесте выставлены одинаковые ключи. А нужны разные.
> …
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>